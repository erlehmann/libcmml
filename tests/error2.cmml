<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml>

<stream id="a1" basetime="npt:100.3" utc="20021222T221005Z">
<import id="a2" granulerate="1000" contenttype="audio/x-wav" src="00-3-10.wav" start="npt:100.3"/>
</stream>

<head lang="en">
<title>
Do we take water for granted?
</title>
<base href="http://www.cmis.csiro.au/cmweb/media/"/>
<meta name="DESCRIPTION" content="A project studying the impact of urban water usage on the environment"/>
<meta name="KEYWORDS" content="Water, Environment"/>
<meta name="TYPE" content="audio interview"/>
<meta name="DATE.PUBLISHED" content="2000-03"/>
<meta name="INTERVIEWEE" content="Mike Young"/>
</head>

<clip id="Introduction" start="npt:x0:01:46.20">
<a href="00-3-18.anx#Introduction">
A way we can store water so that water supplies will not be stressed during hot dry periods</a>
<desc>
Urban water usage research: Australia is a dry continent and urban
communities use a lot of the available water. CSIRO has been working
on an urban water program. The objective is to find out how much water
use is costing the environment and what are the opportunities to
improve its use. Mike Young from CSIRO Land and Water believes we take
water for granted.
</desc>
</clip>

</cmml>
