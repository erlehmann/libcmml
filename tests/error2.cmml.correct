<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml>

<stream id="a1" basetime="npt:100.300" utc="20021222T221005Z">
<import id="a2" granulerate="1000" contenttype="audio/x-wav" src="00-3-10.wav" start="npt:100.300"/>
</stream>

<head lang="en">
<title>
Do we take water for granted?
</title>
<base href="http://www.cmis.csiro.au/cmweb/media/"/>
<meta name="DESCRIPTION" content="A project studying the impact of urban water usage on the environment"/>
<meta name="KEYWORDS" content="Water, Environment"/>
<meta name="TYPE" content="audio interview"/>
<meta name="DATE.PUBLISHED" content="2000-03"/>
<meta name="INTERVIEWEE" content="Mike Young"/>
</head>

