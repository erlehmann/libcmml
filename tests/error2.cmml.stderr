error2.cmml:22:0: Error formatting time
